cpu/cc26xx-cc13xx/Makefile.cc26xx-cc13xx

```diff
@@ -8,6 +8,14 @@ NM      = arm-none-eabi-nm
 SIZE    = arm-none-eabi-size
 SREC_CAT = srec_cat
 
+SREC_TEST = $(shell $(SREC_CAT) --version > null && echo 1)
```

Comment:
On POSIX systems (including Cygwin), that I/O device is named `/dev/null` ; on pure Windows development platforms (e.g., MSYS2), that device is named `NUL:` .